### PR TITLE
Rename doc/howto/cross_compiling into doc/mobile

### LINKED ISSUE #5168: Rename doc/howto/cross_compiling into doc/mobile ?
Would this make the documentation structure cleaner?

### PR DESCRIPTION
**Pull request by Xreki**
- Fix #5168 
- Add mobile documentation to the homepage of documentation.

### CODE REVIEW

**Review by luotao1 — APPROVED** [review]
LGTM！
表格在github页面和网站上都显示的很漂亮

**Comment by Xreki** [comment]
多谢@luotao1
现在，mobile相关编译文档已经能在网站上显示了：
- http://doc.paddlepaddle.org/develop/doc_cn/mobile/index_cn.html
- http://doc.paddlepaddle.org/develop/doc/mobile/index_en.html

File: doc/mobile/cross_compiling_for_android_cn.md [hunk 1]
```diff
@@ -20,10 +20,32 @@ $ docker build -t username/paddle-android:dev . -f Dockerfile.android
 构建好开发镜像后，即可使用开发镜像来编译Android版PaddlePaddle C-API库。
 Android的Docker开发镜像向用户提供两个可配置的参数：
 
-| Argument        | Optional Values         | Default |
-|-----------------|-------------------------|---------|
-|`ANDROID_ABI`    |`armeabi-v7a, arm64-v8a` | `armeabi-v7a` |
-|`ANDROID_API`    |`>= 21` | `21` |
+<table class="docutils">
```
Comment (wangkuiyi) [inline doc/mobile/cross_compiling_for_android_cn.md]: Why do we change above Markdown table format into HTML format?

Comment (luotao1) [reply doc/mobile/cross_compiling_for_android_cn.md]: @jetfuel If we use Markdown table format, can we display it on our website?

Comment (Xreki) [reply doc/mobile/cross_compiling_for_android_cn.md]: @wangkuiyi Because the Markdown table is ugly on the old website `doc.paddlepaddle.org`.

**Pull request merged**